low-level/framework/messaging/egress_accumulated_msgs_processor.py

```diff
@@ -122,7 +128,12 @@ def run(self):
                             continue
                     if "sensor_response_type" in dict_msg["message"]:
                         logger.info(f"Publishing Accumulated Alert: {message}")
-                    self._producer.send([message])
+                    if isinstance(self._producer, MessageProducer):
+                        self._producer.send([message])
+                        logger.error(f"Published Accumulated Message {message}")
```

Comment:
line no. 130 log not enough? also why log.error?

Reply:
Corrected log level, changed it to logger.info.
>>line no. 130 log not enough? 
No this I have added to make sure producer.send() executed successfully.
So if we see this log in log file, then we can say from sspl side we have sent an msg successfully.